src/npc.h

```diff
@@ -390,6 +392,69 @@ enum talk_topic_enum {
  TALK_OLD_GUARD_NEC_COMMO_GOAL,
  TALK_OLD_GUARD_NEC_COMMO_FREQ,
 
+ TALK_RANCH_FOREMAN,//105, Mission source/critical to building up the ranch camp
```

Comment:
That enum isn't really used anymore. (It's only used for loading old saves that still contain numeric talk ids.) Adding entries to it is possible, but useless, they are not used anywhere.